src/webextension/lib/util.js

```diff
@@ -1,3 +1,5 @@
+/* global browser, console */
```

Comment:
You could add an ESLint `env` config of "webextensions" to the .eslintrc file and then not need to define these globals:

```json
  "env": {
    "browser": true,
    "es6": true,
    "node": true,
    "webextensions": true
  },
```